### PR TITLE
Docstring for show_versions in master

### PR DESCRIPTION
**Pull request by MarianD**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by MarianD** [comment]
Connected with issue #33044 and PR #33073.

**Review by jreback — CHANGES_REQUESTED** [review]
this needs an entry in pandas/doc/source/reference/general_functions.py maybe in the Testing section

File: pandas/util/_print_versions.py [hunk 1]
```diff
@@ -88,6 +88,20 @@ def _get_dependency_info() -> Dict[str, JSONSerializable]:
 
 
 def show_versions(as_json: Union[str, bool] = False) -> None:
+    """
+    Provides useful information, important for bug reports.
```
Comment (jreback) [inline pandas/util/_print_versions.py]: remove this first sentence and replace with the 2nd.

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/util/_print_versions.py (hunk 1 above)
Comment (MarianD) [reply pandas/util/_print_versions.py]: @jreback, I originally had it in the form as you suggest, but the 2nd sentence is too long to be in one line, and it is not allowed to have the heading in 2 lines.

**Comment by mroeschke** [comment]
As @MomIsBestFriend mentions https://github.com/pandas-dev/pandas/pull/33073#issuecomment-605607698, you can just update #33073 and merge master into _that_ branch.

Let's continue development in #33073 and address @jreback's comments there

**Pull request closed without merging**